libnavigation-core/src/main/java/com/mapbox/navigation/core/telemetry/MapboxNavigationTelemetry.kt

```diff
@@ -137,7 +138,7 @@ internal object MapboxNavigationTelemetry :
     private var routeProgress: RouteProgress? = null
     private var originalRoute: DirectionsRoute? = null
     private var needStartSession = false
-    private var jobControl: JobControl = ThreadController.getMainScopeAndRootJob()
+    private lateinit var jobControl: JobControl
```

Comment:
Can this `lateinit` be a problem? Iirc there were situations in which `initialize` could be called multiple times.

Reply:
we need that scope just to get location. If we reinit it, it won't affect anything